diff -Naur CGI.pm-3.02/Makefile.PL CGI.pm-3.02-p/Makefile.PL
--- CGI.pm-3.02/Makefile.PL	Fri Jun 15 17:31:23 2001
+++ CGI.pm-3.02-p/Makefile.PL	Fri Jun 22 17:13:41 2001
@@ -4,7 +4,6 @@
 print STDERR "*** WARNING: This is ALPHA SOFTWARE. *** \n";
 print STDERR "*** For production systems, use the CGI 2.X series. ***\n";
 WriteMakefile(
-    'INSTALLDIRS' => 'perl',
     'NAME'	=> 'CGI3',
     'DISTNAME'  => 'CGI.pm',
     'VERSION_FROM'   => 'CGI.pm',
